src/domains/queries.ml

```diff
@@ -156,3 +155,127 @@ struct
     | PartAccess _ -> PartAccessResult.top ()
     | Assert _ -> AR.top ()
 end
+
+type any_query = Any: 'a t -> any_query
+
+module Query = struct
```

Comment:
This module might be better named `Any` or something, because it's not for the actual GADT queries.